wondrous-app/components/Common/KanbanBoard/styles.ts

```diff
@@ -12,7 +12,6 @@ export const KanbanBoardContainer = styled.div`
   margin-top: 32px;
 `;
 export const LoadMore = styled.div`
-  height: 50px;
```

Comment:
if we remove this does the org board / pod boar kanban view has the same load more functionality?

Reply:
@Lamperoyge Yes, it still works.